authui/packages/vite-plugin-authgear-authui/src/index.js

```diff
@@ -342,11 +366,6 @@ function buildPlugin({ input }) {
                 }
                 return null;
               },
-              hashCharacters: "hex",
-              assetFileNames: () => "[name].[hash][extname]",
-              chunkFileNames: () => "[name].[hash].js",
-              entryFileNames: () => "[name].[hash].js",
-              sourcemapFileNames: () => "[name].[chunkhash].map.js",
```

Comment:
Another thought: prefix the hash with some custom prefix like `aghash-[hash]` which we will check in the regex?

Reply:
Actually I want to unset `assetsDir: ""` to use the original behavior of Vite. Then the generated built assets should reside in a separate directory. We do not need the regex at all.